src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -913,7 +914,7 @@ public class SimpleBase
     [KnownType(typeof(SimpleBaseDerived2))]
     public class GenericBase2<T, K>
         where T : new()
-        where K : new()
+    where K : new()
```

Comment:
This is unintentional. I've reverted this change and also done a CodeFormatter pass through serialization code.